Frends.AzureBlobStorage.DownloadBlob/Frends.AzureBlobStorage.DownloadBlob/DownloadBlob.cs

```diff
@@ -8,23 +8,23 @@
 using Azure.Storage.Blobs;
 
 #pragma warning disable CS1591
+#pragma warning disable CS1573
 
 namespace Frends.AzureBlobStorage.DownloadBlob
 {
     public class AzureBlobStorage
     {
         /// <summary>
-        ///     Downloads Blob to a file.
-        ///     See https://github.com/FrendsPlatform/Frends.AzureBlobStorage/tree/main/Frends.AzureBlobStorage.DownloadBlob
+        /// Downloads Blob to a file.
+        /// [Documentation](https://github.com/FrendsPlatform/Frends.AzureBlobStorage/tree/main/Frends.AzureBlobStorage.DownloadBlob)
         /// </summary>
         /// <param name="source"></param>
```

Comment:
Should we write at least some comment on this? That way we can auto-generate some comments for documentation later on. Even the simplest "The definition of source Azure Blob to download".

Reply:
Added parameter information.